packages/sockethub-platform-xmpp/src/index.test.js

```diff
@@ -67,7 +67,17 @@ const job = {
       object: {
         'type': 'message',
         id: 'hc-1234abcd',
-        content: 'hello'
+        content: 'hi all'
+      },
+      target: target.roomuser
+    },
+    correction: {
+      actor: actor,
+      object: {
+        'type': 'message',
+        id: 'hc-1234abcd',
+        replace: { id: 'hc-234bcde' },
```

Comment:
Actually, I think this is `xmpp:replace` in the other direction.

![image](https://user-images.githubusercontent.com/842/148234623-7f916eac-c77f-45ec-85d5-73c58ed21a91.png)

Reply:
Fixed.